testcafe/tests/drive/classification_scenario.js

```diff
@@ -232,7 +231,7 @@ test(`${TEST_MOVE_FOLDER}`, async t => {
     `${FEATURE_PREFIX}/${TEST_MOVE_FOLDER}-1`,
     true
   )
-  await moveMoadal.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
+  await privateDrivePage.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
```

Comment:
Same here, should it use the second parameter?

Reply:
Same answer! :)